storage/src/state/ledger.rs

```diff
@@ -265,22 +115,6 @@ impl<N: Network> LedgerState<N> {
             _ => return Err(anyhow!("Ledger storage state is inconsistent")),
         };
 
-        // If this is new storage, initialize it with the genesis block.
-        if latest_block_height == 0u32 && !ledger.blocks.contains_block_height(0u32)? {
-            let genesis = N::genesis_block();
-
-            // Perform all the associated storage operations as an atomic batch.
-            let batch = ledger.ledger_roots.prepare_batch();
-
-            ledger
-                .ledger_roots
-                .insert(&genesis.previous_ledger_root(), &genesis.height(), Some(batch))?;
-            ledger.blocks.add_block(genesis, Some(batch))?;
-
-            // Execute the pending storage batch.
-            ledger.ledger_roots.execute_batch(batch)?;
-        }
-
```

Comment:
I think this is the only functional change in this PR - since there is `Storage::refresh` that we keep using to catch up to the primary instance of storage (with the reader being the secondary instance), we don't need to insert the genesis block if it's not found; this was removed due to the operations involved being direct writes, and no longer being available to the reader